src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -11,6 +11,11 @@
 
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
 
+    public boolean isMutating(){
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing spaces after `isMutating()`